samples/guestbook/gcp/deploy.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 Google LLC
```

Comment:
Hm. If we have multiple "scripts" like this in the `samples/guestbook/gcp` directory, then the Go compiler is going to complain about redeclaration of `main` function.

Reply:
Ok, I filed https://github.com/golang/go/issues/26848 about this. Meanwhile, we can add `// +build ignore` to scripts like these.

Reply:
aaaand it's rejected.